lib/request-log.js

```diff
@@ -0,0 +1,59 @@
+var format = require('util').format;
```

Comment:
Seems like we are re-inventing the wheel here:
https://github.com/trentm/node-bunyan (see: request serialization)
https://github.com/winstonjs/winston
etc.